test/helper/formatSidechain.js

```diff
@@ -0,0 +1,66 @@
+module.exports = function formatSidechain (txJson) {
+  var txObj = {};
+  txObj.version = txJson.version;
+  txObj.locktime = txJson.locktime;
+
+  txObj.vsc_ccout = txJson.vsc_ccout.map(i => {
+    delete i.scid; // RPC call would be needed for scid
+    return { 
+      ...i, 
+      value: i.value * 1e8, 
+      ftScFee: i.ftScFee * 1e8 ,
```

Comment:
Nice, I like the `1e8` better than `100000000` I used in the `formatCertificate` function. Do you mind changing it there too for consistency?

Reply:
Also nitpick - there is a space before the `,`. Would love to get a linter or prettier in this repo at some point :)

Reply:
yes, agree, linter is a good idea